BAZProjectC1/Components/CarosuelMenu/CarosuelMenuExtension.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CarosuelMenuExtension.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 23/09/22.
+//
+
+import Foundation
+import UIKit
+
+extension CarosuelMenu:UICollectionViewDelegate{
+    
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        optionSelected = indexPath.row
+        delegate?.selectedOption(index: indexPath)
+    }
+}
+extension CarosuelMenu: UICollectionViewDataSource{
```

Comment:
Recuerda los espacios entre llaves
```suggestion
}

extension CarosuelMenu: UICollectionViewDataSource{
```